modules/ckanext-ytp_main/ckanext/ytp/logic.py

```diff
@@ -377,9 +375,7 @@ def action_user_list(context, data_dict):
     if order_by == 'edits':
         query = query.order_by(_desc(
             _select([_func.count(model.Revision.id)],
-                    _or_(
-                        model.Revision.author == model.User.name,
-                        model.Revision.author == model.User.openid))))
+                model.Revision.author == model.User.name)))
```

Comment:
I guess user id comparison is not possible here and multiple users cant have the same name?

Reply:
Just mimicing mainline CKAN implementation https://github.com/ckan/ckan/blob/master/ckan/logic/action/get.py#L858

Reply:
👍